specification/draft/apps.mdx

```diff
@@ -537,6 +537,39 @@ Host behavior:
 * Host SHOULD add the message to the conversation context, preserving the specified role.
 * Host MAY request user consent.
 
+`ui/request-display-mode` - Request host to change display mode
+
+```typescript
+// Request
+{
+  jsonrpc: "2.0",
+  id: 3,
+  method: "ui/request-display-mode",
+  params: {
+    mode: "inline" | "fullscreen" | "pip"  // Requested display mode
+  }
+}
+
+// Success Response
+{
+  jsonrpc: "2.0",
+  id: 3,
+  result: {
+    mode: "inline" | "fullscreen" | "pip"  // Actual display mode set
+  }
+}
+```
+
+Host behavior:
+* Host SHOULD check if the requested mode is in `availableDisplayModes` from host context.
+* If the requested mode is available, Host SHOULD switch to that mode and return it in the response.
```

Comment:
How about `MUST`? App developers will need to rely on this field in cases where they need to adjust the app's view.